### PR TITLE
guides for in app voice

### PR DESCRIPTION
**Pull request by brittBarak**
- Added a guide to [start and receive calls](https://nexmo-developer-pr-1657.herokuapp.com/client-sdk/in-app-voice/guides/start-and-receive-calls)

- Added a guide for [call member actions](https://nexmo-developer-pr-1657.herokuapp.com/client-sdk/in-app-voice/guides/call-member-actions) 

- Removed code snippets from left menu

### CODE REVIEW

**Comment by brittBarak** [comment]
In [Call Member Actions guide](https://nexmo-developer-pr-1657.herokuapp.com/client-sdk/in-app-voice/guides/call-member-actions/android) trying to order the tabs to be:
js | kotlin | java | swift | objC

could you help @tbedford @mheap ?

**Comment by pardel** [comment]
@brittBarak something seems to be amiss with the tabs.
I was expecting, when selecting one language for one snippet, all the other snippets change to that language.
Also, if I refresh the page after selecting the language (ie url is original/url/language), the selection in the left menu is lost.

**Comment by brittBarak** [comment]
@pardel  alright, thanks. sound like more of an NDP thing though. 
CC: @tbedford  @mheap

**Comment by brittBarak** [comment]
@pardel @lukeocodes  please review the content for your platforms, and approve/fix if needed. 
Will open a ticket for NDP team to check the inner tabs selection. It's additional functionality, I believe. But we can start with deploying this for now.

File: _tabbed/guides/start-and-receive-in-app-calls/android.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+### Hang Up
+
+```tabbed_content
+source: '_tutorials_tabbed_content/client-sdk/guides/start-and-receive-in-app-calls-android/hangup'
+frameless: false
+```
+
+### Listen For Call Events
+
+To see updates on the state of the call its member, for example, to know if the other member answered or hung up the call, you should listen to Call events.
```
Comment (lukeocodes) [inline _tabbed/guides/start-and-receive-in-app-calls/android.md]:
```suggestion
To see updates on the state of the call its member, for example, to know if the other member answered or hung up the call, you should listen to Call events.
```

**Comment by brittBarak** [comment]
@lukeocodes @pardel please approve the PR if content-wise it's fine with your platforms.

**Pull request merged**